cmd/ui/main.go

```diff
@@ -116,3 +118,14 @@ func createRedirector(fsys fs.FS, log logrus.FieldLogger) http.HandlerFunc {
 		}
 	}
 }
+
+func proxy(u *url.URL, w http.ResponseWriter, r *http.Request) {
+	proxy := httputil.NewSingleHostReverseProxy(u)
```

Comment:
You should be creating the proxy once, and reusing it, rather than creating a new one for every request.

Reply:
Changed to create the proxy in the `main` block